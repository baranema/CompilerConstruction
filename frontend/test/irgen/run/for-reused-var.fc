extern int printf(char[] format, ...);

int main() {
    int i = 9999;

    printf("for loop:\n");
    for(int i = 0 to 2){
        printf("%d\n", j);
    }

    for(int i = 0 to 2){
        printf("%d\n", j);
    }

    printf("done\n");
    return 0;
}
