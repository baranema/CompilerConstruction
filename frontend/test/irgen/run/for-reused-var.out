for loop:
1
2
1
2
done
